it/itutil/CachingStub.scala

```diff
@@ -30,7 +30,12 @@ trait CachingStub {
   implicit lazy val jsonCrypto = ApplicationCrypto.JsonCrypto
   implicit lazy val encryptionFormat = new JsonEncryptor[JsObject]()
 
-  def stubKeystoreMetadata(session: String, regId: String, companyName: String) = {
+  def stubKeystoreMetadata(session: String,
+                           regId: String,
+                           companyName: String,
+                           completionCapacity: String = "Director"
```

Comment:
Why are we adding Completion Capacity to the test model if it has been removed from Keystore in the code?